src/content/smart-contracts/index.md

```diff
@@ -5,21 +5,15 @@ lang: en
 sidebar: true
 ---
 
-# Introduction to smart contracts {#introduction-to-smart-contracts}
+Smart contracts turn the terms of an agreement into computer code that executes when the contract terms are met.
```

Comment:
> code that executes when the contract terms are met

This is another inaccuracy (which to be fair, existed before your PR). Smart contract code doesn't automatically execute "when the contract terms are met". Contracts still must be "poked" by an EOA in order to execute. e.g. if a loan in MakerDAO becomes undercollateralized (say due to a drop in the market price of ETH), the vault is not automatically liquidated. It requires someone to trigger a function on the smart contract - that's why it's necessary for an entire [incentive structure for "keepers" to fulfill this role](https://docs.makerdao.com/keepers/auction-keepers).

Nitpicking a bit here but I think this is an important aspect of smart contracts that we shouldn't oversell. They're not completely autonomous in the sense that they won't automatically execute logic & state changes UNLESS they are interacted with.

Reply:
This _does_ seem important and I hadn't known that. What would you suggest instead?

Reply:
Made some adjustments here to address these concerns (which I do agree with), and to get this over the finish line.